src/pages/model.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import App from '../../scripts/flux/components/app'
```

Comment:
'import' is only available in ES6 (use esnext option).<br>Missing semicolon.<br>'App' is defined but never used.